src/runtime/program/vote/execute.zig

```diff
@@ -479,6 +482,111 @@ fn updateValidatorIdentity(
     try vote_account.serializeIntoAccountData(VoteStateVersions{ .current = vote_state });
 }
 
+/// Agave https://github.com/anza-xyz/agave/blob/24e62248d7a91c090790e7b812e23321fa1f53b1/programs/vote/src/vote_processor.rs#L121-L131
+fn executeUpdateCommission(
+    allocator: std.mem.Allocator,
+    ic: *InstructionContext,
+    vote_account: *BorrowedAccount,
+    commission: u8,
+) (error{OutOfMemory} || InstructionError)!void {
+    const epoch_schedule = try ic.getSysvar(EpochSchedule);
+    const clock = try ic.getSysvar(Clock);
+
+    try updateCommission(
+        allocator,
+        ic,
+        vote_account,
+        commission,
+        epoch_schedule,
+        clock,
+    );
+}
+
+/// Agave https://github.com/anza-xyz/solana-sdk/blob/69edb584ac17df2f5d8b5e817d7afede7877eded/vote-interface/src/instruction.rs#L415
+fn updateCommission(
```

Comment:
dont think this permalink makes sense - it should be this right? https://github.com/anza-xyz/agave/blob/24e62248d7a91c090790e7b812e23321fa1f53b1/programs/vote/src/vote_state/mod.rs#L743